src/api/app/models/user.rb

```diff
@@ -385,11 +385,13 @@ def update_user_info_from_proxy_env(env)
   #####################
 
   def is_admin?
-    roles.where(title: 'Admin').exists?
+    return @is_admin unless @is_admin.nil?
+    @is_admin = roles.where(title: 'Admin').exists?
```

Comment:
This could also be `@is_admin ||= roles.where(title: 'Admin').exists?`.

Sorry, I didn't see this PR in time before it got merged.

Reply:
@dmarcoux no... if `@is_admin` is false, your code would look for the role again that is exactly what we want to avoid... :unamused: